src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml

```diff
@@ -99,236 +99,236 @@
-    <sys:String x:Key="Activate_Input_CID">Please enter the confirmation ID</sys:String>
+    <sys:String x:Key="Activate_Product_Key.Description">Installa product key di Office, formato della product key: XXXXX-XXXXX-XXXXX-XXXXX-XXXXX.</sys:String>
+    <sys:String x:Key="Activate_Install_Product_Key">_Installa chiave</sys:String>
+    <sys:String x:Key="Activate_Uninstall_Unactivated_Key">_Disinstalla tutte le chiavi non attivate</sys:String>
+    <sys:String x:Key="Activate_Uninstall_All_Product_Keys">Disinstalla _tutte le chiavi</sys:String>
+    <sys:String x:Key="Activate_Install_CID">Installa confirmation ID</sys:String>
+    <sys:String x:Key="Activate_Install_CID.Description">Installa confirmation ID per attivare Office offline.</sys:String>
+    <sys:String x:Key="Activate_Input_CID">Inserire il confirmation ID</sys:String>
 
     <sys:String x:Key="Activate_KMS_Management">Gestione KMS</sys:String>
-    <sys:String x:Key="Activate_KMS_Management.Description">Set or reset the KMS host or port.</sys:String>
-    <sys:String x:Key="Activate_KMS_Apply">_Set host</sys:String>
-    <sys:String x:Key="Activate_KMS_Host">KMS host</sys:String>
-    <sys:String x:Key="Activate_KMS_Host.Description">KMS host name. For example, kms.example.com, kms.example.com:1688 or IP address.</sys:String>
+    <sys:String x:Key="Activate_KMS_Management.Description">Imposta o reimposta l'host o la porta del KMS.</sys:String>
+    <sys:String x:Key="Activate_KMS_Apply">_Imposta host</sys:String>
+    <sys:String x:Key="Activate_KMS_Host">Host KMS</sys:String>
+    <sys:String x:Key="Activate_KMS_Host.Description">Nome host KMS. Per esempio, kms.example.com, kms.example.com:1688 o indirizzo IP.</sys:String>
     <sys:String x:Key="Activate_KMS_Host_Cache.Enable">Abilita la memorizzazione nella cache dell'host</sys:String>
     <sys:String x:Key="Activate_KMS_Host_Cache.Disable">Disabilita la memorizzazione nella cache dell'host</sys:String>
-    <sys:String x:Key="Activate_KMS_Reset">Reset KMS</sys:String>
-    <sys:String x:Key="Activate_KMS_Reset.Description">Remove KMS host name and set KMS port to default.</sys:String>
+    <sys:String x:Key="Activate_KMS_Reset">Reimposta KMS</sys:String>
+    <sys:String x:Key="Activate_KMS_Reset.Description">Rimuove il nome dell'host KMS e imposta la porta KMS predefinita.</sys:String>
 
     <!-- Licenses list -->
-    <sys:String x:Key="Activate_Licenses">Office licenses</sys:String>
-    <sys:String x:Key="Activate_License">Licenses list</sys:String>
-    <sys:String x:Key="Activate_License.Description">Manage installed licenses.</sys:String>
-    <sys:String x:Key="Activate_License_Refresh">_Refresh</sys:String>
-    <sys:String x:Key="Activate_License_Expand">_Expand all</sys:String>
-    <sys:String x:Key="Activate_License_Collapse">_Collapse all</sys:String>
-    <sys:String x:Key="Activate_License_Clear">C_lear list</sys:String>
```

Comment:
It's recommand to change the hotkey to `Aggio_rna`, `Ridu_ci tutto` and `Pu_lisci lista`. Make sure it's same as English if possible.